src/core/main.go

```diff
@@ -114,6 +115,9 @@ func gracefulShutdown(closing, done chan struct{}, shutdowns ...func()) {
 }
 
 func main() {
+	runMode := flag.String("mode", "normal", "The harbor-core container run mode, it could be normal, migrate or skip_migrate, default is normal")
```

Comment:
This is trivial but it's more common the parm uses `-` as separator if the subcommand has multiple words?
For example 
`kubectl port-forwarding` , `kubectl cluster-info`

Reply:
Updated